p9cpu/crates/p9cpu/src/p9cpu.rs

```diff
@@ -0,0 +1,58 @@
+use anyhow::Result;
+use clap::Parser;
+
+#[derive(clap::ValueEnum, Clone, Debug)]
+enum Net {
+    Tcp,
+    Vsock,
+    Unix,
+}
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
+    #[arg(long, value_enum, default_value_t = Net::Tcp)]
+    net: Net,
+
+    #[arg(long, default_value_t = 11200)]
+    port: u32,
+
+    #[arg(long, default_value = "")]
+    namespace: String,
+
+    #[arg(long, default_value_t = false)]
+    tty: bool,
+
+    #[arg(long)]
+    fs_tab: Option<String>,
+
+    #[arg(long, default_value = "/p9cputmp")]
```

Comment:
Done. Changed to `/tmp`. I don't think using `/tmp` is a good idea, since people may also want to bring their local `/tmp` together with `/lib` `/home` to the remote machine. We can discuss this later.

Reply:
I think you are right. The standard from plan 9 is /mnt/term, not sure if that's right either. What do you recommend?

The problem with a network file system mount to a top level directory can cause trouble, I am still not sure what's best here.